lib/gpu/render_pass.h

```diff
@@ -50,42 +52,59 @@ class RenderPass : public RefCountedDartWrappable<RenderPass> {
 
   impeller::StencilAttachmentDescriptor& GetStencilBackAttachmentDescriptor();
 
-  impeller::VertexBuffer& GetVertexBuffer();
-
   impeller::PipelineDescriptor& GetPipelineDescriptor();
 
   bool Begin(flutter::gpu::CommandBuffer& command_buffer);
 
   void SetPipeline(fml::RefPtr<RenderPipeline> pipeline);
 
+  void SetHasIndexBuffer(bool value);
+
+  bool HasIndexBuffer() const;
+
+  void ClearBindings();
+
+  bool Draw();
+
+  using BufferUniformMap =
+      std::unordered_map<const flutter::gpu::Shader::UniformBinding*,
+                         impeller::BufferAndUniformSlot>;
+  using TextureUniformMap =
+      std::unordered_map<const flutter::gpu::Shader::TextureBinding*,
+                         impeller::TextureAndSampler>;
+
+  BufferUniformMap vertex_uniform_bindings;
+  TextureUniformMap vertex_texture_bindings;
+  BufferUniformMap fragment_uniform_bindings;
+  TextureUniformMap fragment_texture_bindings;
```

Comment:
Uniform bindings are wiped out after each `RenderPass::Draw` call. `Command` was hanging on to this state between draws. This is similar to what `Command` was doing previously, except it also resolves https://github.com/flutter/flutter/issues/155335 by tracking maps with unique binding identities.